app/controllers/taxons_controller.rb

```diff
@@ -21,7 +21,7 @@ def redirect_to_www
   def taxon
     @taxon ||= begin
       taxon_base_path = params[:taxon_base_path]
-      Taxon.find('/alpha-taxonomy/' + taxon_base_path)
+      Taxon.find("/" + taxon_base_path)
```

Comment:
It does not seem to work for the root path unfortunately. The controller params are:

```
{"controller"=>"taxons", "action"=>"show", "taxon_base_path"=>"alpha-taxonomy/ca860c58-665d-491d-88ed-26bf2e76fe19-school-education-5-to-18"}
```

Hence the extra `/` at the beginning.